eng/Generate.ps1

```diff
@@ -72,7 +72,12 @@ function Add-CadlRanch-TypeSpec([string]$testName, [string]$projectPrefix, [stri
     $tspMain = Join-Path $cadlRanchFilePath $testName "main.tsp"
     $clientTsp = Join-Path $cadlRanchFilePath $testName "client.tsp"
     $mainTypeSpecFile = If (Test-Path $clientTsp) { Resolve-Path $clientTsp } Else { Resolve-Path $tspMain }
-    Add-TypeSpec "$projectPrefix$testName" $projectDirectory $mainTypeSpecFile "$configString--option @azure-tools/typespec-csharp.new-project=true" "-n"
+    if ($projectPrefix -eq "typespec-nonAzure-") {
```

Comment:
I am fine to use nonAzure for now, but that name is subject to change in the future.